actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -298,11 +297,13 @@ def auto_discovery_link_tag(type = :rss, url_options = {}, tag_options = {})
       #   favicon_link_tag 'mb-icon.png', rel: 'apple-touch-icon', type: 'image/png'
       #   # => <link href="/assets/mb-icon.png" rel="apple-touch-icon" type="image/png" />
       def favicon_link_tag(source = "favicon.ico", options = {})
-        tag("link", {
-          rel: "icon",
-          type: "image/x-icon",
-          href: path_to_image(source, skip_pipeline: options.delete(:skip_pipeline))
-        }.merge!(options.symbolize_keys))
+        ActionView.deprecator.silence do
+          tag("link", {
+            rel: "icon",
+            type: "image/x-icon",
+            href: path_to_image(source, skip_pipeline: options.delete(:skip_pipeline))
+          }.merge!(options.symbolize_keys))
+        end
```

Comment:
Would the following work here?
```suggestion
        tag.link(**{
          rel: "icon",
          type: "image/x-icon",
          href: path_to_image(source, skip_pipeline: options.delete(:skip_pipeline))
        }.merge!(options.symbolize_keys))
```